common/protos/identity/agent.proto

```diff
@@ -14,17 +15,38 @@ service IINAgent {
   // Requesting network unit's state from a foreign IIN agent.
   rpc RequestIdentityConfiguration(NetworkUnitIdentity) returns (common.ack.Ack){}
   // Handling network unit's state sent by a foreign IIN agent.
-  rpc SendIdentityConfiguration(NetworkUnitIdentity) returns (common.ack.Ack){}
-  // user or agent triggers a flow to collect signatures attesting an
-  // external/foreign network unit's state and recording to ledger
-  rpc FlowAndRecordAttestations(NetworkUnitIdentity) returns (common.ack.Ack) {}
+  rpc SendIdentityConfiguration(AttestedMembership) returns (common.ack.Ack){}
   // Requesting attestation from a local IIN agent.
-  rpc RequestAttestation(NetworkUnitIdentity) returns (common.ack.Ack){}
+  rpc RequestAttestation(AttestedSecurityDomain) returns (common.ack.Ack){}
   // Handling attestation sent by a local IIN agent.
-  rpc SendAttestation(NetworkUnitIdentity) returns (common.ack.Ack){}
+  rpc SendAttestation(AttestedSecurityDomain) returns (common.ack.Ack){}
```

Comment:
We planned to send `Attestation` only right in this flow, not whole `AttestedSecurityDomain`?

Reply:
I'm not sure what we agreed on exactly, but upon reflection, I think it won't suffice to just send an `Attestation`. A secondary agent can't simply send an `Attestation` because it is attesting a collection (`AttestedMembershipSet`). So I think we can keep it as is and assume that the `AttestedSecurityDomain` parameter in this service endpoint will only contain the signature of the client.